libyara.NET.nuspec

```diff
@@ -2,34 +2,28 @@
 <package >
   <metadata>
     <id>libyara.NET</id>
-    <version>3.5.2</version>
-    <authors>kallanreed@outlook.com</authors>
-    <owners>kallanreed@outlook.com</owners>
-    <licenseUrl>https://github.com/kallanreed/libyara.NET/blob/master/LICENSE</licenseUrl>
-    <projectUrl>https://github.com/kallanreed/libyara.NET</projectUrl>
+    <version>4.0.1</version>
+    <authors>odspsecurity@microsoft.com</authors>
```

Comment:
please review the MSFT nuget attribute requirements here and ensure that the .nuspec files in this project meet the requirement: https://devdiv.visualstudio.com/DevDiv/_wiki/wikis/DevDiv.wiki?wikiVersion=GBwikiMaster&pagePath=%2FProduct%20Development%20Reference%2FNuGet&pageId=972&anchor=attribute-requirements